src/code.rs

```diff
@@ -555,6 +557,66 @@ fn build_table_fns(
     }}
 "##));
 
+    #[cfg(feature = "advanced-queries")]
+    // Table::filter() helper fn
+    {
+        let diesel_backend = &config.diesel_backend;
+        let filters = table
```

Comment:
this array / iterator could likely be directly pushed onto the buffer, instead of collecting it in a intermediate vec (though not important to change now)